update Observability tech references (#12098)
kyma-project/kyma: docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-observability.md

---
title: Observability charts
---

You find the configurable parameters in charts and subcharts in the respective `values.yaml` files. The `values.yaml` files are fully documented and provide details on the configurable parameters and their customization.

> To override the configuration, redefine the default values in your custom `values.yaml` file and [deploy them](../../04-operation-guides/operations/03-change-kyma-config-values.md).

Reviewer: I'm not pushing 'cause you're describing multiple charts here, but all the other chart and sub-chart documents contain this exact phrase, so I would probably be a good idea to be consistent here. 
![image](https://user-images.githubusercontent.com/53601578/133340674-bdd5c067-eff0-4965-88a4-fea03754304c.png)
```suggestion
To configure the Observability charts, override the default values of the respective `values.yaml` files. This document links to the charts with the parameters that you can configure.

> **TIP:** See how to [change Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md).
```

Author: This sentenced was rephrased during the review. We want to point out that users must not touch the original values.yaml but their custom one. I'll think about a way to phrase this that is more consistent to our template but keeps that distinction.

Author: Actually, I propose to switch the default sentence. The current default text isn't as concise as it could be.

### Monitoring

> [Monitoring values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/values.yaml) and subcharts:

Reviewer: ```suggestion
[Monitoring `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/values.yaml) and subcharts:
```

> - [Grafana values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/grafana/values.yaml)
> - [Kube-state-metrics values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/kube-state-metrics/values.yaml)
> - [Prometheus Istio values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-istio/values.yaml)
> - [Prometheus Node Exporter values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-node-exporter/values.yaml)
> - [Prometheus Pushgateway values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-pushgateway/values.yaml)

Reviewer: ```suggestion
- [Grafana `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/grafana/values.yaml)
- [Kube-state-metrics `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/kube-state-metrics/values.yaml)
- [Prometheus Istio `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-istio/values.yaml)
- [Prometheus Node Exporter `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-node-exporter/values.yaml)
- [Prometheus Pushgateway `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-pushgateway/values.yaml)
```

### Logging

> [Logging values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/values.yaml) and subcharts:
> 
> - [Fluent Bit values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/fluent-bit/values.yaml)
> - [Loki values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/values.yaml)

Reviewer: ```suggestion
[Logging `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/logging/values.yaml) and subcharts:

- [Fluent Bit `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/fluent-bit/values.yaml)
- [Loki `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/values.yaml)
```

### Tracing

> [Tracing values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/tracing/values.yaml)

Reviewer: ```suggestion
[Tracing `values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/tracing/values.yaml)
```

### Kiali

> [Kiali values.yaml](https://github.com/kyma-project/kyma/blob/master/resources/kiali/values.yaml)

Reviewer: ```suggestion
[Kiali `values.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/kiali/values.yaml)
```
